gfx.h

```diff
@@ -480,6 +480,9 @@ ID3D12GraphicsCommandList *gfxGetCommandList(GfxContext context);
 GfxResult gfxSetCommandList(GfxContext context, ID3D12GraphicsCommandList *command_list);
 GfxResult gfxResetCommandListState(GfxContext context); // call this function before returning to gfx after externally modifying the state on the command list
 
+IDXGISwapChain4* gfxGetSwapChain(GfxContext context);
+GfxResult gfxSetSwapChain(GfxContext context, IDXGISwapChain4 *swapchain);
+
```

Comment:
probably should be put down with the other interop functions. Same with the other 3 new functions (and be nice and move the definitions to the new order as well)